.gitignore

```diff
@@ -20,4 +20,13 @@ icons/sprite.svg
 # Bun server
 /binaries/
 
-.wrangler
\ No newline at end of file
+.wrangler
```

Comment:
I'd prefer if we didn't add extra ignores here. You can ignore the files for just your repo by following this suggestion: https://stackoverflow.com/a/653458/4697675